XenAdmin/TabPages/HAPage.cs

```diff
@@ -52,6 +52,12 @@ internal partial class HAPage : BaseTabPage
 
         private IXenObject xenObject;
 
+        private bool newPoolHABox;

```

Comment:
Could we have a better name for this? 'newPoolHABox' doesn't suggest its usage, which is "if this is true I need to restart the initialising timer"
Also, I suggest the camelCasing for the private fields (HBInitTimeElapsed)